### PR TITLE
Authorization token

### PR DESCRIPTION
**Pull request by tisnik**
Implementation of new step to generate authorization token & store it into the context to be used later by tests.

### CODE REVIEW

File: integration-tests/features/steps/common.py [hunk 1]
```diff
@@ -678,3 +685,34 @@ def check_analyzed_dependency(context, package, version):
...
+                'exp': expiry,
+                'iat': datetime.datetime.utcnow(),
+                'sub': userid
+            }
+            token = jwt.encode(payload, key=private_key, algorithm='RS256')
+            decoded = token.decode('utf-8')
+            # print(decoded)
+            context.token = decoded
+        except Exception as e:
+            print("Error occured: " + e)
```
Comment (fridex) [inline integration-tests/features/steps/common.py]: I think the try-except block is not necessary here - you would like to propagate failures on JWT token generation (?).

Comment (tisnik) [inline integration-tests/features/steps/common.py]: Thanks for review. Actually I would not like to propagate failures, I'm updating the code.

**Comment by tisnik** [comment]
[test this]

**Comment by tisnik** [comment]
[test this]

**Comment by tisnik** [comment]
@fridex, WDYT? Thanks in advance for review.

**Comment by tisnik** [comment]
@fridex, WDYT? Thanks in advance for review.

**Pull request merged**